.ado/windows-vs-pr.yml

```diff
@@ -1,4 +1,3 @@
-# This file defines the Windows Visual Studio PR build steps used during the CI loop
```

Comment:
># [](start = 0, length = 1)

I wonder why to remove the comment?

Reply:
It's not really accurate anymore, since we have jobs running macOS, and some not using Visual Studio at all. I also want to rename the file, but that requires some other config changes.